splinterd/tests/framework/network.rs

```diff
@@ -41,15 +44,49 @@ pub enum NetworkNode {
 
 impl Network {
     pub fn new() -> Network {
+        let biome_auth = {
+            #[cfg(feature = "biome-credentials")]
```

Comment:
I think you can just use `cfg!(feature = "biome-credentials")` for this

Reply:
removed